phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java

```diff
@@ -1221,4 +1222,29 @@ public void rvcOffsetTrailingNullKeyTest() throws Exception {
         }
     }
 
+    // Test point lookup over data table with index hint and hinted plan is not applicable
+    @Test
+    public void testRVCOffsetWithNotApplicableIndexHint() throws Exception {
+        String sql = String.format("SELECT /*+ INDEX(%s %s)*/ %s FROM %s "
+                        + "WHERE t_id = 'b' AND k1 = 2 AND k2 = 3 OFFSET (%s)=('a', 1, 2)",
+                TABLE_NAME, INDEX_NAME, TABLE_ROW_KEY,TABLE_NAME,TABLE_ROW_KEY);
+        try (Statement statement = conn.createStatement()){
+            ResultSet rs = statement.executeQuery("EXPLAIN " + sql);
+            String actualQueryPlan = QueryUtil.getExplainPlan(rs);
+            // As hinted plan is not applicable so use data plan which is point lookup
+            assertTrue(actualQueryPlan.contains("POINT LOOKUP ON 1 KEY OVER " + TABLE_NAME));
+        }
+    }
+
+    @Test
+    public void testRVCOffsetWithNotApplicableAndPointLookup() throws Exception {
```

Comment:
is this test applicable to this PR?

Reply:
Yes, I added this test as I am doing changes for point lookup case and wanted to assert that I am not breaking the scenario where its point lookup but no query plan is applicable. Just wanted to be extra sure.